tests/test_multipole/test_multipole.py

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env python
+
+import os
+import sys
+sys.path.insert(0, os.pardir)
+from testing_harness import TestHarness, PyAPITestHarness
+import openmc
+from openmc.stats import Box
+from openmc.source import Source
+
+
+class DistribmatTestHarness(PyAPITestHarness):
```

Comment:
Also, perhaps we should check that OPENMC_MULTIPOLE_LIBRARY is set and if not raise an error saying that it is not. Otherwise, you just get 'results don't match' which is not very helpful.